src/guides/v2.4/graphql/mutations/add-products-to-cart.md

```diff
@@ -0,0 +1,314 @@
+---
+group: graphql
+title: addProductsToCart mutation
+---
+
+Magento 2.4.1 introduced the `addProductsToCart` mutation that streamlines the process of adding the products to the shopping cart. It allows you to add a number of products to the cart at one time regardless of product types.
+
+You must specify the Cart ID along with the list of SKU and Quantity pairs as parameters to add the products to the shopping cart.
+
+## Syntax
+
+```graphql
+mutation {
```

Comment:
As a rule, we have a little bit different look of the syntax example. We skip the "mutation section", you may look at the example [here](https://github.com/magento/devdocs/pull/7775/files?file-filters%5B%5D=.md#diff-9abfbdc0146fcbb83a35601f3b3b3617R12)

Reply:
I see I haven't been consistent about this, but the majority of mutations do include the word `mutation` in the syntax. I left the word in here.

Reply:
Hello @rogyar.
Thank you for your review.

I am not sure, but I think my proposition is correct.
The "Syntax" is different for all topics about mutations here.
Please take a look at:
- https://devdocs.magento.com/guides/v2.4/graphql/mutations/add-bundle-products.html
- https://devdocs.magento.com/guides/v2.4/graphql/mutations/add-downloadable-products.html

I think my proposed syntax will be more understandable for all developers (juniors, middles, etc.)
How do you think about it?

Also, I want to fix it for all mutation topics here, if you'll approve my proposition.

Thank you in advance.

cc @keharper @dmytro-ch